src/main/resources/db.migration/LIB_V2024.03.04.0000__create_tables.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE library_app.author (
+    author_id VARCHAR(36) NOT NULL,
+    first_name VARCHAR(50),
+    last_name VARCHAR(50),
+    biography VARCHAR(250),
+    PRIMARY KEY (author_id)
+);
+
+CREATE TABLE library_app.book_category (
+    book_category_id VARCHAR(36) NOT NULL,
+    name VARCHAR(50),
+    description VARCHAR(250),
+    PRIMARY KEY (book_category_id)
+);
+
+CREATE TABLE library_app.book (
+    book_id VARCHAR(36) NOT NULL,
+    title VARCHAR(50),
+    description VARCHAR(250),
+    year INTEGER,
+    author_id VARCHAR(36) NOT NULL,
+    book_category_id VARCHAR(36) NOT NULL,
+    PRIMARY KEY (book_id),
+    FOREIGN KEY (author_id) REFERENCES author (author_id),
+    FOREIGN KEY (book_category_id) REFERENCES book_category (book_category_id)
+);
+
+CREATE TABLE library_app."user" (
+    user_id VARCHAR(36) NOT NULL,
+    first_name VARCHAR(50),
+    last_name VARCHAR(50),
+    email VARCHAR(50),
+    password VARCHAR(50),
+    user_role VARCHAR(10),
```

Comment:
in `user` table all of fields should not null

Reply:
Done